### DISCUSSION TITLE
Can you help me with environment setup? (macOs)

### DISCUSSION THREAD

**Discussion by EdwinFajardoBarrera** [tchiotludo/akhq#954] [Q&A]
I forked the repo and build the docker image (docker-compose -f docker-compose-dev.yml up -d)

Now docker container is running like this:
![image](https://user-images.githubusercontent.com/36389489/146226223-cfa822ae-51fa-4e4e-ba7f-04a60b55387c.png)

Errors:
akhq_kafka_1: `Command [/usr/local/bin/dub path /etc/kafka/ writable] FAILED !`
akhq_schema-registry_1: 
```
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/usr/share/java/cp-base-new/slf4j-log4j12-1.7.30.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/usr/share/java/cp-base-new/slf4j-simple-1.7.30.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
log4j:WARN No appenders could be found for logger (io.confluent.admin.utils.cli.KafkaReadyCommand).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
```

akhq_connect_1: 
```
[Errno 13] Permission denied: '/etc/kafka/connect-log4j.properties'
Command [/usr/local/bin/dub template /etc/confluent/docker/log4j.properties.template /etc/kafka/connect-log4j.properties] FAILED !
```

Environment: MacOs 12 (Monterey) Macbook pro mid 2015
Docker version 20.10.10
Gradle: 7.3.1
Java OpenJDK 11.0.12

**Comment by tchiotludo** [comment]
Seems that there is an issue with the confluent image, use this image : `image: confluentinc/cp-kafka:7.0.0` for all confluentinc in the docker-compose

**Reply by EdwinFajardoBarrera** [reply]
It's working well now, thank you!

**Comment by EdwinFajardoBarrera** [comment]
Sorry, but I'm getting this error (**akhq_akhq_1**):

![image](https://user-images.githubusercontent.com/36389489/146241181-7c627e77-1db1-4bc8-8658-cb0250ee3bc1.png)

**Reply by tchiotludo** [reply]
humm :thinking: 
Honestly, I don't use anymore this docker to develop. 
I mostly use my ide to start : 
- Webserver: Using `org.akhq.App` on ide like intellijidea 
- React dev server : Using `npm run start`
Docker is only used to get a local kafka / connect / etc ... 
You should try it's better than a docker running all the stuff

**Reply by EdwinFajardoBarrera** [reply]
It was an error on my proxy, thanks for your help!

**Comment by EdwinFajardoBarrera** [comment]
I have this error: "Couldn't find any clusters on your configuration file, please ensure that the configuration file is loaded correctly
". Is required to add anything more? @tchiotludo

**Reply by tchiotludo** [reply]
Yes you should provide a configuration files for your kafka brokers etc .. 
On my side, I add this env var on ide configuration : `MICRONAUT_ENVIRONMENTS=dev` and I add broker configuration in `src/main/resources/application-dev.yml`

**Reply by EdwinFajardoBarrera** [reply]
![image](https://user-images.githubusercontent.com/36389489/146981353-c145c4ae-457b-41e2-8277-9952e65a1f4f.png)

I just added the connections config and env variable but I still having errors:

**Caused by: org.apache.kafka.common.config.ConfigException: No resolvable bootstrap urls given in bootstrap.servers**

I tried with localhost but it doesn't work either.

![image](https://user-images.githubusercontent.com/36389489/146983659-49c0d206-c733-4a87-a8a1-2a0d525c62a8.png)

**2021-12-21 13:03:06,893 WARN  r-thread-2 org.akhq.log.access        [Date: 2021-12-21T13:02:06.631287-06:00] [Duration: 60262 ms] [Url: GET /api/local/topic] [Status: 409] [Ip: /0:0:0:0:0:0:0:1] [User: -]**

I'm running kafka, zookeeper, schema_registry and connect from docker (docker-compose-dev.yml)

![image](https://user-images.githubusercontent.com/36389489/146982930-c2a0dc77-4a15-45cd-a01b-5bfe193700ee.png)

**Reply by tchiotludo** [reply]
Kafka port is `9092` not `9091` (for jmx only)
